### ISSUE TITLE
Sistema de alertas a usuarios

### ISSUE THREAD

**Issue by ealejandro101** [ealejandro101/journal-research-api#8]
Que los usuarios se puedan registrar a alertas de revistas. como las convocatorias

**Comment by jasolutions** [comment]
Correcto, yo me registro y pueda seleccionar las revistas de mi interés para estar informado y tenga alertas de cierres de convocatoria, de nuevo número publicado (si obtenemos esto del ojs)
